validator/sawtooth_validator/server/cli.py

```diff
@@ -118,10 +128,29 @@ def check_directory(path, human_readable_name):
     return errors
 
 
+def _split_comma_append_args(arg_list):
+    new_arg_list = []
+    for arg in arg_list:
+        if ',' in arg:
```

Comment:
split on a string with no commas will just return an array of the input string: you don't need to check for this.